client/packages/programs/src/JsonForms/common/components/Text.tsx

```diff
@@ -130,7 +142,10 @@ const UIComponent = (props: ControlProps) => {
         value: text ?? '',
         sx: { width },
         style: { flexBasis: '100%' },
-        onChange: e => onChange(e.target.value ?? ''),
+        onChange: e =>
+          useDebounce
+            ? onDebounceChange(e.target.value ?? '')
+            : onChange(e.target.value ?? ''),
```

Comment:
As mentioned above, this doesn't actually work for me unless I use `schemaOptions?.useDebounce` here

Reply:
You correctly added the `useDebounce` field to the Zod "Options" type above, but then didn't use it.

Reply:
thanks @CarlosNZ, nice catch, i rushed this out and I thought I tested it, but not properly: 10596a49888d272bd9c06853ea4e16521cfee6f1